randnum.c

```diff
@@ -0,0 +1,23 @@
+/* Yael Kelmer.
```

Comment:
Perfect code! 💯 you did it efficiently and followed all the guidelines! Nice job! 💯